tests/sitemap.test.js

```diff
@@ -20,7 +20,7 @@ const urlset = '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" ' +
 const dynamicUrlSet = '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
 const xmlDef = '<?xml version="1.0" encoding="UTF-8"?>'
 const xmlPriority = '<priority>0.9</priority>'
-const xmlLoc = '<loc>http://ya.ru</loc>'
+const xmlLoc = '<loc>http://ya.ru/</loc>'
```

Comment:
a side-effect of this change is that root domains now end in `/`. seems pretty inconsequential